modin/_compat/methods.py

```diff
@@ -0,0 +1,33 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
```

Comment:
This file should probably be located under _compat/pandas.

Reply:
This is questionable, as `_compat/pandas` is designed for `modin.pandas` (i.e. the functions related to user-facing API) while `_compat/methods.py` is used for Modin internals.